src/lib/vclock/vclock.h

```diff
@@ -374,23 +374,27 @@ vclock_lex_compare(const struct vclock *a, const struct vclock *b)
 
 /**
  * Return a vclock, which is a componentwise minimum (when @sign is -1) or
- * maximum (when @sign is +1) between vclocks @a a and @a b. Do not take
- * vclock[0] into account.
+ * maximum (when @sign is +1) between vclocks @a a and @a b.
  *
  * @param[in,out] a Vclock containing the minimum components.
  * @param b Vclock to compare with.
  * @param sign Which extremum to find:
  *             -1 for minimum,
  *             +1 for maximum.
+ * @param ignore_zero Whether to order by 0-th component or not.
+ *        The 0 component is ignored on anonymous replicas when
```

Comment:
I'd say the 0th vclock component is ignored in replication generally. No only on anonymous replicas.
Every replica has its own 0th vclock component and uses it for local space writes

Reply:
Fixed